contracts/modules/staking/StakingPool.sol

```diff
@@ -210,10 +210,28 @@ contract StakingPool is IStakingPool, ERC721 {
   ) external onlyCoverContract {
     uint length = tokenIds.length;
     for (uint i = 0; i < length; i++) {
-      transferFrom(from, to, tokenIds[i]);
+      operatorTransferFrom(from, to, tokenIds[i]);
     }
   }
 
+  function operatorTransferFrom(address from, address to, uint256 tokenId) internal {
```

Comment:
this is needed as far as i can tell there's no way around it.

Same as in CoverNFT

If there's any easier option to do this i'm open.

Reply:
seems related to https://github.com/NexusMutual/smart-contracts/issues/522